opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -246,21 +246,30 @@ where
 
         #[cfg(feature = "experimental_use_tracing_span_context")]
         if let Some(span) = _ctx.event_span(event) {
+            use opentelemetry::trace::TraceContextExt;
             use tracing_opentelemetry::OtelData;
-            let opt_span_id = span
-                .extensions()
-                .get::<OtelData>()
-                .and_then(|otd| otd.builder.span_id);
-
-            let opt_trace_id = span.scope().last().and_then(|root_span| {
-                root_span
-                    .extensions()
-                    .get::<OtelData>()
-                    .and_then(|otd| otd.builder.trace_id)
-            });
-
-            if let Some((trace_id, span_id)) = opt_trace_id.zip(opt_span_id) {
-                log_record.set_trace_context(trace_id, span_id, None);
+            if let Some(otd) = span.extensions().get::<OtelData>() {
+                if let Some(attributes) = &otd.builder.attributes {
```

Comment:
attaching span attributes to log should not be done by default. Let's discuss this in an issue and come up with a design before adding this.

I think this is a very valid capability, and should be offered by the appender natively (on opt-in basis of course). But we need to design this carefully to avoid duplications.
(Another important question is - do we only append the attributes from the immediate parent Span? What about nested span scenarios?)

Reply:
Sure, I'll remove the feature for now.